test/AppKernel.php

```diff
@@ -11,12 +11,18 @@ class AppKernel extends Kernel
     private $configuration;
     private $enabledPlugins;
 
-    public function __construct(array $configuration, array $enabledPlugins)
+    private $cacheDir;
+    private $logsDir;
+
+    public function __construct(array $configuration, array $enabledPlugins, $uniqueIdentifier, $cacheDir, $logsDir)
     {
-        parent::__construct('test', true);
+        parent::__construct('test' . $uniqueIdentifier, true);
```

Comment:
No problem, it is a monday morning after all ;-)

Since each test we run defines it's own cache, we need this separation. This way the caches don't influence eachother, and the classnames are generated uniquely (well, `mt_rand`-unique).
Because the class names are generated as well, we don't need process isolation.

```
.string(77) "/var/folders/51/zjs4tmks4kz7t5lt6wy6t7n00000gn/T/symfony-phpunit559a31edde41a"
testTest9738465DebugProjectContainer
.string(77) "/var/folders/51/zjs4tmks4kz7t5lt6wy6t7n00000gn/T/symfony-phpunit559a31ef37f2f"
testTest2190819DebugProjectContainer
```